python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -78,6 +78,10 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         "e.g. 0.5 is equal to 2x oversampling",
         default=1,
     )
+    modelSize = pexConfig.Field[int](
+        doc="Internal model size for PIFF",
+        default=25,
```

Comment:
Should this be limited to odd integers only? If not, let's include an even value of `modelSize` in the tests as well.